ph5/clients/ph5toms.py

```diff
@@ -420,14 +472,14 @@ def create_trace(self, station_to_cut, mp=False):
 
             if stc.sample_rate != 0:
                 traces = self.ph5.cut(stc.das, start_time,
-                                      new_endtime,
+                                      stc.endtime,

```

Comment:
Is this the fix for #287?

Reply:
Part of it. There are some lines a bit higher up that calculated new_endtime that were also removed.  Those lines just added 1 sample interval to the end time.